src/tor/configure.ac

```diff
@@ -584,7 +586,6 @@ AC_CHECK_FUNCS(
 	backtrace \
 	backtrace_symbols_fd \
 	eventfd \
-	explicit_bzero \
```

Comment:
Source for this change? Will it be in the tor upstream someday?

Reply:
Please read the comment few lines up for the source of change.

Reply:
> explicit_bzero was removed as it breaks compat with GLIBC 2.21

Explicit_bzero was popping up in the export lists and it was bound to glibc 2.25, which is too new for many distros

Reply:
I saw that. The question is did you take it from somewhere and is there a chance it'll appear in official tor sources? Because it's likely to be overwritten by subsequent tor upgrades

Reply:
I removed it myself. I think the likelihood it'll appear in the parent repo is quite low as they definitely don't need it be compatible with glibc 2.21